crates/pessimistic-proof/src/local_exit_tree/data.rs

```diff
@@ -110,27 +117,33 @@ where
 
     /// Returns the root of the tree.
     pub fn get_root(&self) -> H::Digest {
-        let get_last_layer = |i| self.get(TREE_DEPTH - 1, i);
+        let get_last_layer = |i| {
+            self.layers[TREE_DEPTH - 1]
```

Comment:
I'm worried that this direct indexing could panic at runtime.

Reply:
`layers` is of type `[Vec<H::Digest>; TREE_DEPTH]`, so you shouldn't worry too much :)